workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/fragments/group/liferay-learn/fragments/knowledge-article-featured-group/index.html

```diff
@@ -53,24 +53,26 @@
 
 			[#list queryParams?keys as key]
 				[#assign
-					queryParam = queryParams[key]
+					value = queryParams[key]
 
-					queryParts = queryParts + ["${key?url}=${queryParam?url}"]
+					queryParts = queryParts + ["${key?url}=${value?url}"]
 				/]
 			[/#list]
 
-			[#assign knowledgeArticles = restClient.get("/c/p2s3knowledgearticles/?" + queryParts?join('&')) /]
+			[#assign recentArticles = restClient.get("/c/p2s3knowledgearticles/?" + queryParts?join('&')) /]
```

Comment:
Hi @anabxalves 
'Articles' can be any type of article. Let's specify.
change it to `recentKnowledgeArticles`

Reply:
Also remember to update where it's being called, and to update it on UAT.